### PR TITLE
Assignment 1

### PR DESCRIPTION
**Pull request by jstanley6**
https://zealous-pond-0771e721e.3.azurestaticapps.net/

### CODE REVIEW

**Comment by grahamna** [comment]
Don't forget to provide the URL to your Azure Deployment!

**Comment by tienv9** [comment]
Everything seems to be correct and there doesn't seem to be any problem, good job.

Create a Vue app ✅

Uses TypeScript ✅
Add Vuetify ✅
Uses Vitest ✅
Create a simple page called game.vue ✅

Page should have a card with a heading that with a clever name for the game ✅
some default placeholder body text ✅
and a button to take you back to the home page ✅
there should be a button on the home page to access this page ✅
Check all the code into your repo fork ✅

Create a static web app in Azure ✅

Create the container where your GitHub repo will deploy
Link the site with your GitHub repo
Update the CI/CD pipeline ✅

Pipeline runs on each check in
Vitest run and pass ✅
Vue app builds ✅
Vue app deploys to Azure ✅
Azure site works ✅

Post your Azure URL in your readme.md file. Replace the URL in the readme with your URL. ✅
Extra Credit
Come up with up to three unique ideas for stories we could implement as part of our project. Post these in your readme.md. ❌
One point per idea up to three points

File: wordle-web/src/pages/game.vue [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<template>
```
Comment (mmwoodfo) [inline wordle-web/src/pages/game.vue:1]: This file should be under your views, which is basically just your pages folder.

File: wordle-web/src/pages/game.vue [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  <v-card
+    class="mx-auto"
+    max-width="344"
+    variant="outlined"
+  >
+    <v-card-item>
+      <div>
```
Comment (mmwoodfo) [inline wordle-web/src/pages/game.vue:8]: You have alot of divs here

**Pull request closed without merging**